app/mailers/user_mailer.rb

```diff
@@ -97,8 +97,7 @@ def account_reset_cancel(email_address)
     mail(to: email_address.email, subject: t('user_mailer.account_reset_cancel.subject'))
   end
 
-  def please_reset_password(email_address, message)
-    @message = message
+  def please_reset_password(email_address)
```

Comment:
There used to be a freeform `message` param in the old templates, I decided to remove it as part of the cleanup